tests/cypress/integration/case_5_image_rotate.js

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+/// <reference types="cypress" />
+
+context('Check if the image is rotated', () => {
+
+    const caseId = '5'
+    const labelName = `Case ${caseId}`
+    const taskName = `New annotation task for ${labelName}`
+    const attrName = `Attr for ${labelName}`
+    const textDefaultValue = 'Some default value for type Text'
+    const imageFileName = `image_${labelName.replace(' ', '_').toLowerCase()}`
+    const image = `${imageFileName}.png`
+    const width = 800
+    const height = 800
+    const posX = 10
+    const posY = 10
+    const color = 'gray'
+    function rotateClockwise() {
+        cy.get('.cvat-canvas-container')
+        .trigger('keydown', {key: 'Control'})
+        .trigger('keydown', {key: 'r'})
+        .trigger('keyup', {key: 'r'})
+        .trigger('keyup', {key: 'Control'})
+    }
+    function rotateAnticlockwise() {
+        cy.get('.cvat-canvas-container')
+        .trigger('keydown', {key: 'Control'})
+        .trigger('keydown', {key: 'Shift'})
+        .trigger('keydown', {key: 'r'})
+        .trigger('keyup', {key: 'r'})
+        .trigger('keyup', {key: 'Control'})
+        .trigger('keyup', {key: 'Shift'})
```

Comment:
Here we can leave shortcuts. So, in this case we testing both the scenarious

Reply:
Combined it into one function with a condition.